_typos.toml

```diff
@@ -0,0 +1,40 @@
+[type.xml]
+extend-glob = ["*.xml"]
+check-file = false
+
+[type.jsonl]
+extend-glob = ["*.jsonl"]
+check-file = false
+
+[type.list]
+extend-glob = ["*.list"]
+check-file = false
+
+[type.csv]
+extend-glob = ["*.csv"]
+check-file = false
+
+[type.minjs]
+extend-glob = ["*.min.js"]
+check-file = false
+
+[files]
+extend-exclude = [
+    # auto-generated files
+    "package-lock.json",
+
+    # submodules
+    "src/browser/*",
```

Comment:
It might be good to actually run this on the submodules so we can find typos even if typo checking isn't configured in the submodule itself. (I am thinking of this as a check before publishing a new release.)

Reply:
It makes sense to skip speckjs though

Reply:
Sure, I was imagining the submodules would have their own typo checking, but if we wanna just do it here that'd be fine. We'd just have to check for typos when we do a submodule update and fix separately (or ignore in short term).